proto/prysm/v1alpha1/validator-client/interface.go

```diff
@@ -0,0 +1,6 @@
+package validatorpb
+
+// SignRequest_Object exports the private isSignRequest_Object interface.
+type SignRequest_Object interface {
```

Comment:
Why use a snake case here? is it because it's next to pb?

```suggestion
type SignRequestObject interface {
```

Reply:
yeah, I didn't htink about it much besides it was the same format as the private thing. I'll rename